### PR TITLE
17 - OS Integration

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by rachelandrew** [comment]
@jpmedley I can't see the preview for this, which should be at https://deploy-preview-7720--web-dev-staging.netlify.app/learn/pwa/os-integration/

**Review by rachelandrew — CHANGES_REQUESTED** [review]
There's something odd happening as I can't se the preview and it's missing from the nav.

File: src/site/content/en/learn/pwa/os-integration/index.md [hunk 1]
```diff
@@ -1,11 +1,259 @@
 ---
-title: OS integration
-description: >
-  Our app now works outside of the browser. In this chapter, you will understand how to integrate further with the operating system.
+layout: post
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply src/site/content/en/learn/pwa/os-integration/index.md:2]: Removed.

**Comment by jpmedley** [comment]
>There's something odd happening as I can't se the preview and it's missing from the nav.

I'm guessing it has something to do with the recent changes to the order of the files. I'm going to close this and submit a fresh PR.

**Pull request closed without merging**